Added Dockerfile and dummy script

Slavko12345 opened the pull request:
Added Dockerfile and dummy train script.

Slavko12345 committed:
Added Dockerfile and dummy script

truskovskiyk commented on ml-service/Dockerfile:
```diff
@@ -0,0 +1,5 @@
+FROM python:3.9.7
+
+COPY train.py /app/
+RUN pip install numpy scikit-learn
```
please move installation before 

```
COPY train.py /app/
```
reasons:  invalidating the Docker build cache
see full explanation: https://stackoverflow.com/questions/34398632/docker-how-to-run-pip-requirements-txt-only-if-there-was-a-change

truskovskiyk replied:
please add versions and put deps to requirements.txt 

reference: https://learnpython.com/blog/python-requirements-file/

truskovskiyk reviewed (commented):
@Slavko12345 looks good! please address feedback

also - could you add CI yaml file to this PR too?

Slavko12345 committed:
Fixed Dockefile. Added requirements. Added CI yaml

Slavko12345 commented:
Added requirements, and fixed order of lines in the dockerfile. 
I never thought it is possible to make so many mistakes in literally 3 lines of code, thanks)

truskovskiyk reviewed (approved):
@Slavko12345 amazing!
